src/Operation/FindAndModify.php

```diff
@@ -49,6 +50,9 @@ class FindAndModify implements Executable
      *
      * Supported options:
      *
+     *  * arrayFilters (document array): A set of filters specifying to which
+     *    array elements an update should apply.
```

Comment:
This comment should be consistent with what we have in `FindOneAndUpdate`. I think it's just missing the "not supported" paragraph.